tests/ci/workflow_jobs_lambda/app.py

```diff
@@ -126,6 +126,20 @@ def send_event_workflow_job(workflow_job: WorkflowJob) -> None:
         )
 
 
+def killed_job(wf_job: dict) -> bool:
+    """a hack to identify the killed runner if "Complete job" is omit"""
+    if (
+        wf_job.get("status", "") != "completed"
+        or wf_job.get("conclusion", "") != "failure"
+    ):
+        # The task either success or in progress
+        return False
+    return not any(
+        step["name"] == "Complete job" and step["conclusion"] is not None
+        for step in wf_job["steps"]
+    )
```

Comment:
it's a bit hard to follow the logic, maybe
```suggestion
    return any(
        step["name"] == "Complete job" and step["conclusion"] is None
        for step in wf_job["steps"]
    )
```

Reply:
"Complete job" is GH's step and we assume it's always there, right?